Assets/MixedRealityToolkit/_Core/Inspectors/Utilities/Lines/DataProviders/SplineDataProviderInspector.cs

```diff
@@ -0,0 +1,336 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+using Microsoft.MixedReality.Toolkit.Internal.Utilities;
+using Microsoft.MixedReality.Toolkit.Internal.Utilities.Lines.DataProviders;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEditorInternal;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Inspectors.Utilities.Lines.DataProviders
+{
+    [CustomEditor(typeof(SplineDataProvider))]
+    public class SplineDataProviderInspector : BaseMixedRealityLineDataProviderInspector
```

Comment:
Summary Tag

Reply:
Also, loads of empty spaces in this script

Reply:
Not sure what you mean by empty spaces. They're there intentionally to make it more organized and easier to read.